spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/domain/entities/Application.java

```diff
@@ -44,7 +49,24 @@
 
 	@lombok.Builder(builderClassName = "Builder", toBuilder = true)
 	private Application(String name, @Nullable BuildVersion buildVersion, @Nullable String status,
-			@Nullable Instant statusTimestamp, List<Instance> instances) {
+			@Nullable Instant statusTimestamp, List<Instance> instances, String groupName) {
+		Assert.notNull(name, "'name' must not be null");
```

Comment:
`instances` and `groupName` are also NonNull parameters, and only `name` is asserted. What is the criterion for assert checking?

Reply:
Thank you for the Reminder! I have added some Assert checking now.